src/app/gps/qgsgpsinformationwidget.cpp

```diff
@@ -626,15 +669,28 @@ void QgsGPSInformationWidget::displayGPSInformation( const QgsGPSInformation &in
   }
 
   QgsPointXY myNewCenter;
+  nmeaPOS myNewNmeaPosition;
```

Comment:
We're avoiding introducing new variables using the old "my" prefix - can you remove please?